docs/sample-apps/multimodal-chat/backend/README.md

```diff
@@ -0,0 +1,48 @@
+# Pixel Table Backend API
+
+FastAPI-based backend service for Pixel Table providing multimodal processing capabilities.
+
+## API Routes
+
+### Core Endpoints
+- `GET /` - Service info
+- `GET /health` - Health check
+- `GET /api/files` - List all uploaded files
+
+### File Management
+- `POST /api/upload` - Upload document/video files
+- `POST /api/videos/upload` - Upload video files
+- `POST /api/audio/upload` - Upload audio files
+- `GET /api/videos` - List uploaded videos
```

Comment:
Not sure this GET/API/VIDEOs is used in ChatItnerface.tsx anymore? I searched for it. It seems like api/files cover both use cases?

Reply:
Removed `GET /api/videos`